code/game/machinery/vending/vendor_types/crew/commanding_officer.dm

```diff
@@ -71,6 +71,7 @@ GLOBAL_LIST_INIT(cm_vending_clothing_commanding_officer, list(
 
 		list("COMBAT EQUIPMENT (TAKE ALL)", 0, null, null, null),
 		list("Commanding Officer's M3 Armor", 0, /obj/item/clothing/suit/storage/marine/MP/CO, MARINE_CAN_BUY_ARMOR, VENDOR_ITEM_MANDATORY),
+		list("Commanding Officer's M3 Armor", 0, /obj/item/clothing/suit/storage/marine/MP/CO/jacket, MARINE_CAN_BUY_ARMOR, VENDOR_ITEM_MANDATORY),
```

Comment:
Give different name in vendor for new armor

Reply:
Fixed